ion/binaryreader.go

```diff
@@ -212,7 +213,12 @@ func (r *binaryReader) next() (bool, error) {
 
 		// If it's a local symbol table, install it and keep going.
 		if r.ctx.peek() == ctxAtTopLevel && isIonSymbolTable(r.annotations) {
-			err := r.readLocalSymbolTable()
+
+			st, err := readLocalSymbolTable(r, r.cat)
+			if err == nil {
```

Comment:
I might be misunderstanding something. This conditional is `true` when there is no error present? We'd then set `r.lst = st` and `return `false, nil`.

Should this conditional follow the same error handling pattern as above? `err != null`, then this conditional is `true` if an error is detected and then `return false, err`.? Otherwise, `r.lst = st` and `return true, nil`?

Reply:
> Otherwise, r.lst = st and return true, nil?

When we import a local symbol table, we will continue consuming the next data in the stream incase there are more local symbol tables to import, therefore we return `false,nil`